### PR TITLE
Metadata editor styling

### PR DESCRIPTION
**Pull request by mazameli**
Just some nice little styling tweaks: https://www.loom.com/share/061a17a1ea284b6fa58c3c3b49fcf2e5

### CODE REVIEW

File: frontend/src/metabase/components/EditBar.jsx [hunk 1]
```diff
@@ -30,6 +31,7 @@ class EditBar extends Component {
         )}
       >
         <div>
+          <Icon name="pencil" size={12} className="text-white mr1" />
```
Comment (kulyk) [inline frontend/src/metabase/components/EditBar.jsx:34]: Just wanted to double-check: the `EditBar` is a shared component that's also used in dashboard and admin permissions edit modes. I see you removed the "You're editing ..." part from the dataset editor, but the "You're editing this dashboard" label is in place. Is that intended?

**Review by kulyk — APPROVED** [review]
Looks nice, thank you! 🙏  Just wanted to double-check the EditBar change

File: frontend/src/metabase/components/EditBar.jsx (hunk 1 above)
Comment (mazameli) [reply frontend/src/metabase/components/EditBar.jsx:34]: Yep, I did know that, and did check how it looked on dashboards, but now you're making me question why I ended up not removing the "You're editing" text there on dashboards. Let me look at it again.

Comment (mazameli) [reply frontend/src/metabase/components/EditBar.jsx:34]: I think it'll look funny 'cause the title of the dashboard is there below the edit bar:

image

**Pull request merged**